source/cards/deck.py

```diff
@@ -32,6 +32,9 @@ def shuffle(self):
     def get_random_card(self):
         return self.cards.pop()
 
+    def get_random_cards(self, amount=1):
+        return [self.cards.pop() for _ in xrange(amount)]
+
     def get_hand(self):
         return Hand([self.cards.pop() for _ in xrange(HAND_INITIAL_SIZE)])
 
```

Comment:
Now that we have the 'get_random_cards' function, this should return 
**Hand(self.get_random_cards(amount = HAND_INITAL_SIZE))**